storage/rocksdb/ha_rocksdb.cc

```diff
@@ -13992,6 +13987,15 @@ int ha_rocksdb::external_lock(THD *const thd, int lock_type) {
       }
     }
 
+    /*
+      It's nice to do the following on start of every statement. The problem
+      is, handler->start_stmt() is not called for INSERTs.
+      So, we put this code here.
+    */
+    Rdb_transaction *const tx =
```

Comment:
do we need these code  for Line 13968 `bool unsafe_for_binlog = THDVAR(thd, unsafe_for_binlog);`